docroot/themes/humsci/humsci_basic/src/scss/elements/_tables.scss

```diff
@@ -0,0 +1,52 @@
+.hb-table-wrap {
+  margin: auto;
+  overflow-x: auto;
+}
+
+table {
+  $border-color: #d9d7d2;
```

Comment:
you can use: `darken($su-color-driftwood,  15%);` to get this color

Reply:
Ended up using `lighten()` based on what was in Figma.